addons/account/views/account_journal_views.xml

```diff
@@ -30,6 +30,11 @@
             <field name="arch" type="xml">
                 <form string="Account Journal">
                     <sheet>
+                        <div name="button_box" class="oe_button_box">
+                            <button class="oe_stat_button" type="action"
+                                    name="%(action_move_line_form)d" icon="fa-book" string="Journal Entries"
+                                    context="{'search_default_journal_id':active_id}"/>
+                        </div>
```

Comment:
This is a redirection to `account.move`. What I see in the task is "view group by of AML..."; that would be with the views deriving from `account.view_move_line_tree_grouped`

Reply:
changed it to `action_account_moves_all_a`